.eslintignore

```diff
@@ -0,0 +1,7 @@
+lib
+coverage
+.nyc_output
+node_modules
+# we want to ignore "tests/files" here, but unfortunately doing so would
+# interfere with unit test and fail it for some reason.
+# tests/files
```

Comment:
See also https://github.com/benmosher/eslint-plugin-import/issues/1413#issuecomment-531471382 :
> There is a technical problem: Since we don't want to lint `tests/files`, an `.eslintignore` file needs to be added. But doing that alters the behavior of unit tests for `no-unused-modules`, the rule will consider `tests/files/**/*` to be ignored, hence failing the tests.